readme.md

```diff
@@ -44,7 +44,7 @@ Yields:
 
 ## API
 
-### visit([node](https://github.com/wooorm/unist#unist-nodes)\[, type\], callback\[, reverse\])
+### visit([node](https://github.com/wooorm/unist#unist-nodes)\[, type], callback\[, reverse])
```

Comment:
Note that I’d first just apply this removal of escapes to those shortcuts: `[foo]` only, `[foo][]` and `[foo][bar]` can be kept escaped (at first, at least, we can dig into that later).